pngcheck.c

```diff
@@ -5323,6 +5323,12 @@ FIXME: add support for decompressing/printing zTXt
       printf("%s  file doesn't end with a%sEND chunk\n", verbose? "":fname,
         mng? " M":"n I");
       set_err(kMinorError);
+    } else {
+      if (have_mDCV && !have_cICP) {
+        printf("%s  file has %smDCV%s, but does not have %scICP%s\n", verbose? "":fname,
+        color? COLOR_YELLOW:"", color? COLOR_NORMAL:"", color? COLOR_YELLOW:"", color? COLOR_NORMAL:"");
+      set_err(kMinorError);
```

Comment:
Could you please fix the indentation? I can understand that `set_err` should be under the if-statement, but if it's not indented properly, then it looks as if it's misplaced around the subsequent curly bracket.